src/library.js

```diff
@@ -1582,6 +1582,10 @@ LibraryManager.library = {
   llvm_floor_f64: 'Math_floor',
   llvm_round_f32: 'Math_round',
   llvm_round_f64: 'Math_round',
+  llvm_minnum_f32: 'Math_min',
+  llvm_minnum_f64: 'Math_min',
+  llvm_maxnum_f32: 'Math_max',
```

Comment:
the `maxnum` ones seem to have already existed (line 4375). Your addition is in a better location, though, let's remove the old ones.

Reply:
Not sure how I missed those! Updated, thanks.